client/src/components/ACOAdditionalData/styles.module.scss

```diff
@@ -0,0 +1,6 @@
+:local(.inputsContainer) {
+  :global(.MuiFormControl-root) {
```

Comment:
Can you explain please why do we should put this line: `:global(.MuiFormControl-root)`

Reply:
Targeting an element with `.MuiFormControl-root` somewhere inside an element with `.inputsContainer`